plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx

```diff
@@ -110,26 +132,31 @@ const renderTabs = ( contextValue: MarketplaceContextType ) => {
 const Tabs = ( props: TabsProps ): JSX.Element => {
 	const { additionalClassNames } = props;
 	const marketplaceContextValue = useContext( MarketplaceContext );
-	const { setSelectedTab } = marketplaceContextValue;
+	const { selectedTab, setSelectedTab } = marketplaceContextValue;
+	const [ activeTabs, setActiveTabs ] = useState( getActiveTabs( '' ) );
 
 	const query: Record< string, string > = useQuery();
+	const queryLoaded = Object.keys( query ).length > 0;
 
 	useEffect( () => {
 		if ( query?.tab && tabs[ query.tab ] ) {
 			setSelectedTab( query.tab );
-		} else {
+		} else if ( queryLoaded ) {
 			setSelectedTab( DEFAULT_TAB_KEY );
 		}
-	}, [ query, setSelectedTab ] );
+	}, [ query, queryLoaded, setSelectedTab ] );
 
+	useEffect( () => {
+		setActiveTabs( getActiveTabs( selectedTab ) );
```

Comment:
Same with my previous comment, I find `active` slightly confusing. Maybe `visibleTabs` is a good alternative?

Reply:
I see how this name is confusing. Updated in [b17fd74](https://github.com/woocommerce/woocommerce/pull/40303/commits/b17fd7432e3f767d4f1b43b906e2006ab6b5c3ec)